ftw/file/locales/fr/LC_MESSAGES/ftw.file.po

```diff
@@ -93,6 +93,11 @@ msgstr "Vous êtes sur le point de remplacer le fichier existant. <br />Appuyez
 msgid "download"
 msgstr "télécharger"
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```

Comment:
French translation:
Téléchargement bloqué. Le logiciel malveillant ${name} a été trouvé dans le fichier.

Please ask Niklaus or Andres if this is good French 😉